recipes/wagtail-markdown/meta.yaml

```diff
@@ -31,7 +31,6 @@ test:
 about:
   home: https://github.com/torchbox/wagtail-markdown
   license: zlib
-  license_family: zlib
```

Comment:
There is no `license_family` matching `zlib`. So was just easier to drop `license_family` entirely as it is not required.